### PR TITLE
refactor dubbo decode.

### PR DESCRIPTION
**Pull request by zonghaishang**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: pkg/protocol/xprotocol/dubbo/command.go [hunk 1]
```diff
@@ -30,10 +30,10 @@ type Header struct {
 	Id      uint64
 	DataLen uint32
 
-	Event           int // 1 mean ping
-	TwoWay          int // 1 mean req & resp pair
-	Direction       int // 1 mean req
-	SerializationId int // 2 mean hessian
+	IsEvent         bool // 1 mean ping
```
Comment (nejisama) [inline pkg/protocol/xprotocol/dubbo/command.go:33]: 注释没有同步调整

Comment (zonghaishang) [reply pkg/protocol/xprotocol/dubbo/command.go:33]: 已经对齐了，github显示问题

Comment (cch123) [reply pkg/protocol/xprotocol/dubbo/command.go:33]: 是不是没 go fmt 哦

Comment (cch123) [reply pkg/protocol/xprotocol/dubbo/command.go:33]: @zonghaishang ，改成 bool 了，应该是 true means ping 吧

File: pkg/protocol/xprotocol/dubbo/command.go [hunk 2]
```diff
@@ -30,10 +30,10 @@ type Header struct {
...
+	IsTwoWay        bool // 1 mean req & resp pair
```
Comment (cch123) [inline pkg/protocol/xprotocol/dubbo/command.go:34]: 同上

File: pkg/protocol/xprotocol/dubbo/command.go (hunk 1 above)
Comment (zonghaishang) [reply pkg/protocol/xprotocol/dubbo/command.go:33]: event和ping是2码事情

File: pkg/protocol/xprotocol/dubbo/command.go (hunk 2 above)
Comment (zonghaishang) [reply pkg/protocol/xprotocol/dubbo/command.go:34]: event代表2种事件，一种是heartbeat，另外一种是readonly事件。 之前同学理解ping不全对，他这个理解应该是heartbeat.

**Comment by cch123** [comment]
LGTM
